configuration/items.md

```diff
@@ -322,8 +324,8 @@ The following guidelines apply to user-added icon files:
 openHAB can work with either Bitmap (`png`) or Vector (`svg`) icon files.
 The format should match the display capabilities of the user interfaces in use (e.g. Basic UI).
 It is thereby important to decide on one format beforehand; vector graphics are recommended.
-The setting can be done via Paper UI or inside the configuration files `classicui.cfg` and `basicui.cfg` located in `$OPENHAB_CONF/services`.
-Note that image files in formats other than `png` and `svg` will be ignored.
+The setting can changed via Paper UI for most user interfaces, please check the user interface documentation if in doubt.
+Note that image files with the wrong file ending will be ignored.
```

Comment:
With this change I've tried to address the issue from a more abstract pov. There are other UIs besides Classic UI and Basic UI.